aws/v12.7.0/README.md

```diff
@@ -0,0 +1,39 @@
+# :zap: Giant Swarm Release v12.7.0 for AWS :zap:
```

Comment:
```suggestion
# :zap: Tenant Cluster Release v12.7.0 for AWS :zap:
```

To adapt to https://github.com/giantswarm/giantswarm/issues/14339

Reply:
Good catch! I forgot about it. I updated the PR. Please, have another look.